plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/server/TestProject.java

```diff
@@ -87,7 +87,7 @@ private void createJavaProject(String projectName) throws CoreException, JavaMod
     if (!Strings.isNullOrEmpty(containerPath)) {
```

Comment:
I know I'd find it a bit easier to read through this if you moved this check into `addContainerPathToRawClasspath()`.  It simplifies this method, plus it moves the state-checking into the method that's actually using the state and reduces the implicit context required to read through `addContainerPathToRawClasspath()`.